### PR TITLE
:rocket:[TAWS-33] Initial file structure and code for Contact Form

### PR DESCRIPTION
**Pull request by lax0248917**
This is just the basic file structure and code for a Contact Form. The styling is just for fun and based on the Home Page and About Us page.

### CODE REVIEW

**Comment by lax0248917** [comment]
Had to adjust some of the responsive styling, added a commit with the changes

File: source/pages/contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="../../styles/style.css">
+	<link rel="stylesheet" href="./styles/contact-us.css">
+	<title>Contact Us</title>
+</head>
+<body>
+	<div class="container">
+		<form action="">
+			<div class="section">
+				<label for="name" id="name">Name</label>
```
[the remark this answers is not in the document]
Comment (mariushilleke) [reply source/pages/contact-us/contact-us.html:14]: Maybe the **_duplicate-id_** question could be checked with a small JavaScript-addition for the contact-us-page by using the DOM-API to log something on the console or anything dynamic 😋

[the remark this answers is not in the document]
Comment (lax0248917) [reply source/pages/contact-us/contact-us.html:14]: To help me understand, are you saying that with Javascript we query the html for duplicate IDs, then have it log any it finds in the console?

File: source/pages/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+			<div class="section">
+				<label for="email" id="email">Email</label>
+				<input class="item" type="email" id="email" placeholder="Email address">
+			</div>
+			<div class="section">
+				<label for="request" id="request">How may we help you?</label>
+				<textarea class="item" name="request" id="request" cols="30" rows="10" placeholder="Please enter your question here..."></textarea>
+			</div>
+			<div class="section">
+				<button class="item button btn-primary" type="submit" id="SubmitBtn" style="width:80px;height:40px;">Submit</button>
```
[the remark this answers is not in the document]
Comment (lax0248917) [reply source/pages/contact-us/contact-us.html:26]: If you like the looks of this one then by all means, but either way I think having a "**Submit**" button class would be great.

File: source/pages/contact-us/contact-us.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariushilleke) [reply source/pages/contact-us/contact-us.html:14]: Actually I do not know whether those IDs are really duplicated if they are contained in functional HTML-tags like `input` or `label` so I would suggest the JavaScript functions or executions just as a short test for that question.
@lax0248917 do you have some free time for a small coding session, maybe something like 30min? Around 11am EST?

[the remark this answers is not in the document]
Comment (lax0248917) [reply source/pages/contact-us/contact-us.html:14]: @mariushilleke I am available, I am assuming 11am EST Saturday the 11th?

[the remark this answers is not in the document]
Comment (mariushilleke) [reply source/pages/contact-us/contact-us.html:14]: Yup :), would be 5pm

[the remark this answers is not in the document]
Comment (lax0248917) [reply source/pages/contact-us/contact-us.html:14]: @mariushilleke do you feel after our coding session that your questions regarding usage of unique identifiers was answered?

**Pull request merged**